### DISCUSSION TITLE
lambda nodejs problem

### DISCUSSION THREAD

**Discussion by Hodhem** [serverless/serverless#9436] [Q&A]
I deployed my zip file which includes index.js and node_modules file and keys for aws thing from awsiot but the lambda function keep showing 
errorType": "Runtime.HandlerNotFound",
  "errorMessage": "index.handler is undefined or not exported",
  "trace": [
    "Runtime.HandlerNotFound: index.handler is undefined or not exported",
    "    at Object.module.exports.load (/var/runtime/UserFunction.js:144:11)",
    "    at Object.<anonymous> (/var/runtime/index.js:45:30)",
    "    at Module._compile (internal/modules/cjs/loader.js:778:30)",
    "    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)",
    "    at Module.load (internal/modules/cjs/loader.js:653:32)",
    "    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)",
    "    at Function.Module._load (internal/modules/cjs/loader.js:585:3)",
    "    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)",
    "    at startup (internal/bootstrap/node.js:283:19)",
    "    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)"
  ]
}

**Comment by pgrzesik** [comment]
Hello @Hodhem - when it comes to support question, please create discussion or use our forum (forum.serverless.com). This tracker is intended for bugs/enhancement proposals. I will move it to discussions section.
